aws/policy/compute.yaml

```diff
@@ -41,6 +41,7 @@ Statement:
         autoscaling:InstanceTypes:
           - t3.nano
           - t3.micro
+          - t2.nano
```

Comment:
Aha! There's 2 things we need to change here actually.  The change you have is good, but also the resource it's acting on is `arn:aws:autoscaling:us-east-1:966509639900:autoScalingGroup:85d03fe6-7fdf-4005-bc5d-06e1d11dc5b7:autoScalingGroupName/ansible-test-37463066-lab-asg`.  In 
https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2autoscaling.html it shows the resource that `UpdateAutoScalingGroup` can be restricted to is `autoScalingGroup*` but we have `autoScalingGroup:*` (notice the wildcard is after a `:` in our policy, which means it won't catch `autoScalingGroupName/*` resources).

Can you please edit the resource to be `'arn:aws:autoscaling:{{ aws_region }}:{{ aws_account_id }}:autoScalingGroup*'`? (with no `:` before the `*`)